docs/imputation.md

```diff
@@ -110,7 +120,7 @@ along with the cumulative coverage, which is the proportion of reads mapping to
 
 ### Required Parameters
 
-**--path-keyfile** or **--read-file** but not both. **--key-file** is the name and path of a keyfile that contains a list of the 
+**--path-keyfile** or **--read-file** but not both. **--path-keyfile** is the name and path of a keyfile that contains a list of the 
 read mapping files or read files (fastq). The keyfile must have two columns labeled sampleName and filename. 
```

Comment:
Capitalize all letters in "fastq" to "FASTQ"; Add backticks to "sampleName" and "filename"